src/signed-xml.ts

```diff
@@ -59,18 +59,23 @@ export class SignedXml {
   /**
    * Specifies the data to be signed within an XML document. See {@link Reference}
    */
-  private references: Reference[] = [];
   private id = 0;
   private signedXml = "";
   private signatureXml = "";
   private signatureNode: Node | null = null;
   private signatureValue = "";
   private originalXmlWithIds = "";
+  private keyInfo: Node | null = null;
+
+  /**
+   * Contains the references that were signed. See {@link Reference}
```

Comment:
nit -
```suggestion
   * Contains the references that were signed. 
   * @see {@link Reference}
```

Reply:
I made this change and others like it in the code. Thanks for pointing this out.